src/integrationTest/java/com/mongodb/kafka/connect/mongodb/MongoDBHelper.java

```diff
@@ -28,11 +28,22 @@
 import com.mongodb.client.MongoClients;
 import com.mongodb.client.MongoDatabase;
 
+import org.testcontainers.mongodb.MongoDBAtlasLocalContainer;
+
 public class MongoDBHelper
     implements BeforeAllCallback, BeforeEachCallback, AfterEachCallback, AfterAllCallback {
   private static final String DEFAULT_URI = "mongodb://localhost:27017";
   private static final String URI_SYSTEM_PROPERTY_NAME = "org.mongodb.test.uri";
   private static final String DEFAULT_DATABASE_NAME = "MongoKafkaTest";
+  private static final String MONGO_USERNAME = "test_username";
+  private static final String MONGO_PASSWORD = "test_password";
```

Comment:
This should be in vault imo. Also, this is a public repo

Reply:
Since this is a docker instance and not an actual mongodb instance -> it should be okay to define these parameters in the code.